code/__DEFINES/species_clothing_paths.dm

```diff
@@ -1,19 +1,8 @@
-//HUMAN PATHS
-///The dmi for humanoid uniforms
-#define DEFAULT_UNIFORM_FILE 'icons/mob/clothing/under/default.dmi'
-///The dmi for humanoid shoes
-#define DEFAULT_SHOES_FILE 'icons/mob/clothing/feet.dmi'
-///The dmi for humanoid oversuits
-#define DEFAULT_SUIT_FILE 'icons/mob/clothing/suits/default.dmi'
-
 //MONKEY PATHS
 ///The dmi for monkey uniforms
 #define MONKEY_UNIFORM_FILE 'icons/mob/species/monkey/uniform.dmi'
 
 //DIGITIGRADE PATHS
-///The dmi containing digitigrade uniforms
```

Comment:
why are you removing these comments and the defines are staying?

Reply:
i remove when im in home